### ISSUE TITLE
Missing stargazers count

### ISSUE THREAD

**Issue by tajo** [vaffel/react-components#30]
http://react-components.com/component/react-portal

- there should be 97 stars and not 0

What's wrong?

**Comment by dinorastoder** [comment]
I see that API returns:  **starCount: 0,** 
http://react-components.com/api/components/react-portal

**Comment by tajo** [comment]
Is that from npmjs.org? Can I fix it somehow?

**Comment by dinorastoder** [comment]
It seams that is a problem with cron job **fetch-stars.js**
https://github.com/vaffel/react-components/blob/master/app/api/github-api.js#L31
https://github.com/vaffel/react-components/blob/master/cron/fetch-stars.js#L32

Github API is returning correct info **stargazers_count: 101,**
https://api.github.com/repos/tajo/react-portal

**Comment by thedamon** [comment]
Yep all the stars counts seem to be 0 as far as I can see. Would it make sense to switch to  to the GitHub api?
